File: matiasmasso/Mat.NET/DTO/DTODocRpt.cs
using System.Collections.Generic;

namespace DTO
{
    public class DTODocRpt
    {
        public Estilos Estilo { get; set; }
        public FuentePapel Papel { get; set; }
        public List<DTODoc> Docs { get; set; }

        public enum Estilos
        {
            Comanda,
            Albara,
            Factura,
            Proforma
        }

        public enum FuentePapel
        {
            Copia,
            Original
        }

        public enum Ensobrados
        {
            No,
            Sencillo
        }

        public DTODocRpt(Estilos oEstilo, FuentePapel oPapel = FuentePapel.Copia) : base()
        {
            Estilo = oEstilo;
            Papel = oPapel;
            Docs = new List<DTODoc>();
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTODocRpt.vb
Public Class DTODocRpt
    Property Estilo As Estilos
    Property Papel As FuentePapel
    Property Docs As List(Of DTODoc)

    Public Enum Estilos
        Comanda
        Albara
        Factura
        Proforma
    End Enum

    Public Enum FuentePapel
        Copia
        Original
    End Enum

    Public Enum Ensobrados
        No
        Sencillo
    End Enum

    Public Sub New(ByVal oEstilo As Estilos, Optional ByVal oPapel As FuentePapel = FuentePapel.Copia)
        MyBase.New()
        _Estilo = oEstilo
        _Papel = oPapel
        _Docs = New List(Of DTODoc)
    End Sub

End Class
